Sources/Commons/Either.swift

```diff
@@ -63,3 +63,15 @@ extension Either: Codable where L: Codable, R: Codable {
         }
     }
 }
+
+extension Either: CustomStringConvertible {
```

Comment:
Sorry, didn't catch. Why we need this `Either` abstraction?

Reply:
It's used for the JSONRPC identifier, which, according to the specs, can be either a `String` or an `Int`